docs/tutorials/share-data-and-events-between-custom-functions-and-the-task-pane-tutorial.md

```diff
@@ -1,158 +1,18 @@
-
-1. Find the **\<Page\>** element. Then change the source location from **Functions.Page.Url** to **Taskpane.Url**.
-
-    ```xml
-    <AllFormFactors>
-    ...
-    <Page>
-      <SourceLocation resid="Taskpane.Url"/>
-    </Page>
-    ...
-    ```
-
-1. Find the `<FunctionFile ...>` tag and change the `resid` from **Commands.Url** to  **Taskpane.Url**.
-
-    ```xml
-    </GetStarted>
-    ...
-    <FunctionFile resid="Taskpane.Url"/>
-    ...
-    ```
-
-1. Save the **manifest.xml** file.
-
-## Configure the webpack.config.js file
-
-The **webpack.config.js** will build multiple runtime loaders. You need to modify it to load only the shared runtime via the **taskpane.html** file.
-
-1. Open the **webpack.config.js** file.
-1. Go to the `plugins:` section.
-1. Remove the following `functions.html` plugin if it exists.
-
-    ```javascript
-    new HtmlWebpackPlugin({
-        filename: "functions.html",
-        template: "./src/functions/functions.html",
-        chunks: ["polyfill", "functions"]
-      })
-    ```
-
-1. Remove the following `commands.html` plugin if it exists.
-
-    ```javascript
-    new HtmlWebpackPlugin({
-        filename: "commands.html",
-        template: "./src/commands/commands.html",
-        chunks: ["polyfill", "commands"]
-      })
-    ```
-
-1. If you removed the `functions` or `commands` plugin, add them as `chunks`. The following JavaScript shows the updated entry if you removed both `functions` and `commands` plugins.
-
-    ```javascript
-      new HtmlWebpackPlugin({
-        filename: "taskpane.html",
-        template: "./src/taskpane/taskpane.html",
-        chunks: ["polyfill", "taskpane", "commands", "functions"]
-      })
-    ```
-
-1. Save your changes and rebuild the project.
-
-    ```command&nbsp;line
-    npm run build
-    ```
-
-    > [!NOTE]
-    > You can also remove the **functions.html** and **commands.html** files. The **taskpane.html** loads the **functions.js** and **commands.js** code into the shared runtime via the webpack updates you just made.
-
-1. Save your changes and run the project. Ensure that it loads and runs with no errors.
-
-   ```command&nbsp;line
-   npm run start
-   ```
-
-## Share state between custom function and task pane code
-
-Now that custom functions run in the same context as your task pane code, they can share state directly without using the **Storage** object. The following instructions show how to share a global variable between custom function and task pane code.
+The following instructions show how to share a global variable between custom function and task pane code. This tutorial assumes that you've completed the [Excel custom functions tutorial](excel-tutorial-create-custom-functions.md), with a **Excel Custom Functions using a Shared Runtime** project using the script type **JavaScript**. Use the add-in you created in that tutorial to complete the following instructions.
 
 ### Create custom functions to get or store shared state
```

Comment:
The step at line 199 can be removed (adding the script to include functions.js in the html file). Since yo office uses webpack this already happens automatically.

Reply:
Fixed! Thanks